src/extra/vtkread.jl

```diff
@@ -154,7 +158,7 @@ function _vtpconnec(vtk)
   end
 
   # construct connectivity elements
-  [connect(ind, Ngon) for ind in inds]
+  [connect(ind) for ind in inds]
```

Comment:
In 3D we may need to specify the geometry type in connect. The Ngon is the default for 2D because we can use the orientation of vertices. But in 3D we need to specify which type we want.